### DISCUSSION TITLE
Добавить раздел про "боли"

### DISCUSSION THREAD

**Discussion by kxnkxv** [doka-guide/content#3325] [Идеи] [answered]
Добавить раздел в котором можно будет описывать решение типовых задач с которыми сталкивается новичек

**Comment by solarrust** [comment, marked as answer]
Привет и тут!
У нас прямо сейчас есть два формата, отлично подходящие под это. 
Короткий формат «На практике», где можно рассказать о применении технологии в реальных условиях, для реальных задачек.
Длинный формат «Рецепты». Его мы создали специально для детального разбора типовых или очень интересных задач на стыке технологий.
Мы будем очень рады идеям для обоих этих форматов, если у тебя прямо сейчас что-то очень болит и надо этим поделиться. 🙌
